aggregator/src/test/java/com/vmware/flowgate/aggregator/MessageProcessingTest.java

```diff
@@ -494,26 +496,6 @@ public void testOpenManageJobExecute1() throws JobExecutionException, JsonProces
       openmanageJob.execute(null);
    }
 
```

Comment:
why remove those code ?

Reply:
Some code submitted after I first pull the flowgate. I will check and add them.